<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputpanel</class>
 <widget class="QWidget" name="inputpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item row="4" column="7">
    <widget class="QPushButton" name="panelButton_0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>48</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QPushButton" name="panelButton_d">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>D</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>68</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QPushButton" name="panelButton_b">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>66</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="panelButton_a">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>A</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>65</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QPushButton" name="panelButton_c">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>67</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="panelButton_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>54</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="panelButton_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>53</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="panelButton_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>52</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="panelButton_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>55</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="panelButton_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>56</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="panelButton_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>57</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="panelButton_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>51</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="panelButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>50</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="panelButton_1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>49</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QPushButton" name="panelButton_e">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>E</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>69</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QPushButton" name="panelButton_f">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>70</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="2" column="11">
    <widget class="QPushButton" name="panelButton_minu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>48</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="4" column="11">
    <widget class="QPushButton" name="panelButton_star">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>*</string>
     </property>
     <property name="buttonValue" stdset="0">
      <char>
       <unicode>42</unicode>
      </char>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="panelButton_lock">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resouce.qrc">
       <normaloff>:/image/resouce/unlocked.png</normaloff>
       <normalon>:/image/resouce/locked.png</normalon>:/image/resouce/unlocked.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="panelButton_del">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>del</string>
     </property>
     <property name="keycode" stdset="0">
      <number>16777223</number>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QPushButton" name="panelButton_backspace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>BackS
&lt;----</string>
     </property>
     <property name="keycode" stdset="0">
      <number>16777219</number>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QPushButton" name="panelButton_pad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>填充</string>
     </property>
     <property name="keycode" stdset="0">
      <number>16777223</number>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QPushButton" name="panelButton_expan">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="keycode" stdset="0">
      <number>16777223</number>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QPushButton" name="closeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>60</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>inputpanel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_lock</sender>
   <signal>toggled(bool)</signal>
   <receiver>inputpanel</receiver>
   <slot>setLock(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_a</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_b</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_c</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_d</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_e</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_f</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_minu</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_expan</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelButton_star</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setLock(bool)</slot>
 </slots>
</ui>
